src/ui/app/spa.css

```
@@ -61,3 +61,63 @@
+ * the meshui binary by `go:embed all:dist` and is expected to run somewhere
+ * with no route to the public internet. Vite fingerprints these into
+ * dist/assets/ and rewrites the URLs below to match, so the embed picks them
+ * up with no manifest to maintain. __tests__/dashboard-fonts.test.ts holds
+ * that: it builds, then opens the files the emitted CSS points at.
+ *
+ * font-display: swap matches the demo, and is the right trade for a font
+ * served from the same origin as the page — first paint is never blocked, and
+ * the swap window is short enough that it lands with the first render.
+ */
+@font-face {
+  font-family: "Geist";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)